### DISCUSSION TITLE
Trigram search

### DISCUSSION THREAD

**Discussion by Lythaeon** [PostgREST/postgrest#3144] [Q&A] [answered]
I can't figure out how I am able to search with trigram through postgrest. Is the operator % even available from the postgrest api in some way or another or is the only way around to use rpcs?

**Comment by steve-chavez** [comment, marked as answer]
> Is the operator % even available from the postgrest api in some way or another or is the only way around to use rpcs?

Not available, but you can do a feature request. RPC only for now.
